### ISSUE TITLE
Add Pipfile

### ISSUE THREAD

**Issue by kiview** [TUnbehaun/phue-racing-flags#1]
In order to make the build easily reproducible for others, I'd recommend adding something like [pipenv](https://pipenv.pypa.io/en/latest/).

If you give me the list of required packages (only some Hue package?), I'd be happy to raise a PR and also add a CI job for building the exe. What did you use for building the distributable?

**Comment by TUnbehaun** [comment]
This is a great idea! I am using a venv locally and was kind of in rush when creating the repo yesterday, so I just uploaded the main files without the dependencies. And I was also planning to make the build process and especially the distributable more transparent.

The pip packages required are:

- phue
- PySimpleGui

For building the distributable I used PyInstaller.

Thanks a lot for your help! Much appreciated.
